libglusterfs/src/inode.c

```diff
@@ -955,17 +986,23 @@ inode_find(inode_table_t *table, uuid_t gfid)
     pthread_mutex_lock(&table->lock);
     {
         inode = __inode_find(table, gfid, hash);
```

Comment:
Why do you need to hash (line 984) if it's the root?

Reply:
Nice catch.